### DISCUSSION TITLE
How do I verify records history

### DISCUSSION THREAD

**Discussion by keenhon** [codenotary/immudb#2004] [Q&A]
I am exploring immudb.

I tried select (history of table) and can see the revision.

I read that immudb is cryptographically verifiable.

I would like to be able to show that results returned from a query to be verified. Example: query select * from (history of employees) where id=1

I have read the document, maybe I missed something. Help would really be appreciated.

**Comment by ostafen** [comment]
Hey, @keenhon, to verify the records returned by an SQL query you have to call the `VerifiableSQLGet` call for each row that you want to verify, using some client SDK.

**Reply by keenhon** [reply]
Any example on it? All I found was

VerifiableSQLGet (VerifiableSQLGetRequest) returns (VerifiableSQLEntry)

How does a SQLGetRequest look like?

**Reply by ostafen** [reply]
You are looking to the GRPC spec, but you should rather use some client SDK which already wraps the GRPC endpoint and does verification for you on the returned proof. Which language/client SDK are you using for executing SQL statements?

**Reply by keenhon** [reply]
I am using node.js
